src/diffusers/pipelines/pixart_alpha/pipeline_pixart_alpha.py

```diff
@@ -237,6 +274,7 @@ def __init__(
 
         self.vae_scale_factor = 2 ** (len(self.vae.config.block_out_channels) - 1)
         self.image_processor = VaeImageProcessor(vae_scale_factor=self.vae_scale_factor)
+        self.model_token_max_length = model_token_max_length
```

Comment:
Can't we set it to the pipeline call args?

Reply:
if it is not something people adjust for each generation, probably ok to be added here
but you would not need to assign it as attribute manually, you can access with `config.model_token_max_length`
```suggestion
```